lms/djangoapps/courseware/tabs.py

```diff
@@ -0,0 +1,55 @@
+"""
+This module is essentially a broker to xmodule/tabs.py -- it was originally introduced to
+perform some LMS-specific tab display gymnastics for the Entrance Exams feature
+"""
+from django.conf import settings
+
+from courseware.access import has_access
+from milestones.api import get_course_milestones_fulfillment_paths
+from student.models import CourseEnrollment
+from xmodule.tabs import CourseTabList
+
+
+def get_course_tab_list(course, user):
+    """
+    Retrieves the course tab list from xmodule.tabs and manipulates the set as necessary
+    """
+    user_is_enrolled = user.is_authenticated() and CourseEnrollment.is_enrolled(user, course.id)
+    xmodule_tab_list = CourseTabList.iterate_displayable(
+        course,
+        settings,
+        user.is_authenticated(),
+        has_access(user, 'staff', course, course.id),
+        user_is_enrolled
+    )
+
+    # Entrance Exams Feature
+    # If the course has an entrance exam, we'll need to see if the user has not passed it
+    # If so, we'll need to hide away all of the tabs except for Courseware and Instructor
+    entrance_exam_mode = False
+    if settings.FEATURES.get('ENTRANCE_EXAMS', False) and settings.FEATURES.get('MILESTONES_APP', False):
+        if getattr(course, 'entrance_exam_enabled', False):
```

Comment:
I've seen this in a few places, but since 'entrance_exam_enabled' is an xBlock field, it'll always be an attribute so you don't have to do a getattr()